src/Spice86.Shared/Interfaces/IMainWindowViewModel.cs

```diff
@@ -6,9 +6,8 @@
 /// GUI of the emulator.<br/>
 /// Displays the content of the video ram (when the emulator requests it) <br/>
 /// Communicates keyboard and mouse events to the emulator <br/>
-/// This is the MainWindowViewModel.
 /// </summary>
-public interface IGui {
+public interface IMainWindowViewModel {
```

Comment:
I feel an interface should be named for what it _does_, not for what it _is_. So this rename makes no sense to me.

Reply:
Reversed.